pkg/cmd/roachtest/tests/node_helpers.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2025 The Cockroach Authors.
```

Comment:
Super nit: `javascript_helpers.go` or something like that? `node` might be confused for a cockroach node.